libraries/install_helpers.rb

```diff
@@ -35,6 +35,8 @@ def version_checksum_algorithm(version)
         'sha512'
       when -> (v) { Gem::Requirement.new('~> 9.0.10').satisfied_by?(v) }
```

Comment:
Couldn't this just be >= 9.0.10 is sha512?

Reply:
🤦‍♂️ Yup